content/rs/administering/database-operations/alerting.md

```diff
@@ -5,29 +5,18 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-In the *Alerts* section of the Redis Enterprise Software database page,
-you can designate which database events trigger alert
-notifications.
+You can select alerts to receive notifications about database status.
+You can also select alerts on cluster status in the [cluster alerts]]({{< relref "/rs/administering/cluster-operations/settings/alerts.md" >}}).
 
-For instructions on configuring alerts at the cluster level, refer to
-[Managing cluster
-alerts]({{< relref "/rs/administering/cluster-operations/settings/alerts.md" >}}).
+Some alerts can only be turned on or off, such as [Periodic backup failed]({{< relref "/rs/administering/database-operations/database-backup.md" >}}).
+Other alerts require that you set a threshold that triggers the notifiction, such as **Dataset size has reached a certain percentage of its limit**.
 
-Some alerts can only be turned on or off, such as **[Periodic
-backup]({{< relref "/rs/administering/database-operations/database-backup.md" >}})
-failed**. Some alerts require setting a threshold, such as **Dataset
-size has reached a certain percentage of its limit**.
+Configured alerts appear in the database **Status** field, in the **Log** page.
+The notifications can also be sent by **email** to users according to their [user settings]
+({{< relref "/rs/administering/designing-production/security/account-management.md#adding-a-user" >}}).
 
-Configured alerts appear in the database **Status **field, in
-the **Log **page, and can also be sent by **email**.
-
-**To enable receiving email alerts**:
+To enable an email alert:
 
 1. Select the checkbox at the bottom of the section.
-1. Add the relevant users on the **Team** page, and make sure that the
-    checkbox Email Alerts is selected (for additional details, refer to
-    [Account
-    Management]({{< relref "/rs/administering/designing-production/security/account-management.md" >}}).
-1. Configure the email server settings on the **General** page (for
-    additional details, refer to [general
-    settings]({{< relref "/rs/administering/cluster-operations/settings/_index.md" >}})).
+1. Add the relevant users on the **Team** page add the database to their [email alerts]({{< relref "/rs/administering/designing-production/security/account-management.md" >}}).
+1. Configure the [email server settings]({{< relref "/rs/administering/cluster-operations/settings/_index.md" >}})).
```

Comment:
Maybe we should first ask the user to configure the mail server setting, then turn on the Email Alert in the bottom of the Cluster Alerts settings page?